src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -0,0 +1,44 @@
+import { qS, qSA, removeChildNodes } from './helpers';
+
+export default function createAdditionalInfoWidget() {
+  const widgets = qSA(document, '.additional-info-container');
+
+  if (widgets.length) {
+    widgets.forEach(el => {
+      const titleNode = qS(el, '.additional-info-title');
+      const titleText =
+        (titleNode && titleNode.textContent) || 'More information';
+      const contentContainer = qS(el, '.additional-info-content').parentNode;
+      const contentContainerID = contentContainer.id;
+      const contentMarkup = qS(el, '.additional-info-content').innerHTML;
+
+      const template = `
+          <button type="button" class="additional-info-button va-button-link" aria-controls="${contentContainerID}" aria-expanded="false">
+            <span class="additional-info-title">${titleText}
+              <i class="fa fa-angle-down"></i>
+            </span>
+          </button>
+          <span id="${contentContainerID}">
+            <div class="additional-info-content">
+              ${contentMarkup}
+            </div>
+          </span>`;
+
+      removeChildNodes(el);
+
+      el.insertAdjacentHTML('afterbegin', template);
```

Comment:
Wow, I had to look up `afterbegin`. I don't think I've ever actually used `insertAdjacentHTML` before!